utils/tests_fetcher.py

```diff
@@ -1215,6 +1127,35 @@ def parse_commit_message(commit_message: str) -> Dict[str, bool]:
         return {"skip": False, "no_filter": False, "test_all": False}
 
 
+JOB_TO_TEST_FILE = {
+    "torch_and_tf": r"tests/models/.*/test_modeling_(?=tf_|[^flax]).*",
+    "torch_and_flax": r"tests/models/.*/test_modeling_(?=flax_|[^tf]).*",
+    "tf": r"tests/models/.*/test_modeling_tf_.*",
+    "torch": r"tests/models/.*/test_modeling_[^flax_|^tf_)].*",
+    "tokenization": r"tests/models/.*/test_tokenization.*",
+    "examples_torch": r"examples/pytorch/.*test_.*",
+    "examples_tensorflow": r"examples/tensorflow/.*test_.*",
+    "examples_flax": r"examples/flax/.*test.*",
+    "exotic_models": r"tests/models/.*(?=layoutlmv|nat|deta|udop|nougat).*",
+    "custom_tokenizers": r"tests/models/.*/test_tokenization_(?=bert_japanese|openai|clip).*",
+    "repo_utils": r"tests/repo_utils/test.*",
+    "pipeline_tf": r"tests/models/.*/test_modeling_tf_.*",
+    "pipeline_torch": r"tests/models/.*/test_modeling__[^flax_|^tf_)].*",
+    "hub": r"tests/models/.*/test_modeling_[^flax_|^tf_)].*",
+    "onnx": r"tests/models/.*/test_modeling_(?=tf_|[^flax]).*",
+}
```

Comment:
We miss the test files like `tests/models/auto/test_image_processing_auto.py`, `tests/models/pix2struct/test_processor_pix2struct.py` etc